nsot/api/views.py

```diff
@@ -670,25 +670,55 @@ def addresses(self, request, pk=None, site_pk=None, *args, **kwargs):
         """Return a list of addresses for this Interface."""
         interface = self.get_resource_object(pk, site_pk)
         addresses = interface.addresses.all()
-
         return self.list(request, queryset=addresses, *args, **kwargs)
 
     @detail_route(methods=['get'])
     def assignments(self, request, pk=None, site_pk=None, *args, **kwargs):
         """Return a list of information about my assigned addresses."""
         interface = self.get_resource_object(pk, site_pk)
         assignments = interface.assignments.all()
-
         return self.list(request, queryset=assignments, *args, **kwargs)
 
     @detail_route(methods=['get'])
     def networks(self, request, pk=None, site_pk=None, *args, **kwargs):
         """Return all the containing Networks for my assigned addresses."""
         interface = self.get_resource_object(pk, site_pk)
-
         return self.list(request, queryset=interface.networks, *args, **kwargs)
 
     @detail_route(methods=['get'])
+    def parent(self, request, pk=None, site_pk=None, *args, **kwargs):
+        """Return parent of interface"""
+        interface = self.get_resource_object(pk, site_pk)
+        parent = [] if interface.parent is None else [interface.parent]
+        return self.list(request, queryset=parent, *args, **kwargs)
```

Comment:
Parent should return only a single object vs. a list. See `NetworkViewSet.parent` for a good reference on how it's done for network objects.